site/docs/hosting/firebase.md

```diff
@@ -0,0 +1,186 @@
+# Hosting: Firebase Functions
+
+This tutorial will guide you through the process of deploying your bot to [Firebase](https://firebase.google.com) using [Firebase Serverless Functions](https://firebase.google.com/docs/functions).
```

Comment:
```suggestion
This tutorial will guide you through the process of deploying your bot to [Firebase Functions](https://firebase.google.com/docs/functions).
```

Reply:
I would keep the `Firebase` part, since it gives a link to the actual firebase website